src/state/swap/hooks.test.ts

```diff
@@ -26,7 +26,7 @@ describe('hooks', () => {
         queryParametersToSwapState(parse('?outputCurrency=invalid', { parseArrays: false, ignoreQueryPrefix: true }))
```

Comment:
above

`'?inputCurrency=ETH&outputCurrency=0x6b175474e89094c44da98b954eedeac495271d0f&exactAmount=20.5&exactField=outPUT',`

is `outPUT` supposed to have that casing?